dev/com.ibm.ws.cdi.beansxml_fat/fat/src/com/ibm/ws/cdi12/fat/tests/EnablingBeansXmlValidationTest.java

```diff
@@ -56,7 +56,7 @@ public static void setUp() throws Exception {
      * Test to ensure that an old beans.xml which was parsed by OWB is rejected as long as validation is enabled.
      */
     @Test
-    @SkipForRepeat(SkipForRepeat.EE8_FEATURES)
+    @SkipForRepeat({SkipForRepeat.EE8_FEATURES, SkipForRepeat.EE9_FEATURES})
     @AllowedFFDC({ "org.jboss.weld.exceptions.IllegalStateException", "com.ibm.ws.container.service.state.StateChangeException" }) //We are expecting these errors, but we're using Allowed because the schema download occasionally fails.
```

Comment:
This `@SkipForRepeat` should have a comment explaining why it's not tested on EE8/9.

I.e. is it just that it's not worth repeating, or is it not valid on 8/9?

Reply:
I'll let @benjamin-confino answer that one, I don't know

Reply:
Weld changed some of it's output so we have to versions of this test. One for CDI 1.2 and one for CDI 2+

The other method has the explaining comment: `//In CDI 2.0 with weld 3.0.4 or later WELD-001210 is a warning not an error. `